make.js

```diff
@@ -120,7 +120,7 @@ function getTaskList(taskList) {
 
     taskList.forEach((taskName) => {
```

Comment:
Please check with "node make.js build --task AzureCliV2" - if it doesn't AzureCLIV2, that is blocking